README.md

```diff
@@ -132,8 +147,31 @@ Send build environment variables is set. Collecting the following information:
   'RUN_DISPLAY_URL' => 'http://127.0.0.1:9090/job/mabl%20integration/47/display/redirect'
 ```
 
+### Upgrading from previous versions
+
+#### Upgrading from pre-0.0.20 versions
+
+Note that
+* Jenkins versions 1.x are no longer supported
+* Minimum required Java version is Java 8
+* Jobs with mabl steps will have to be _manually_ updated
+  1. Create a Secret text kind credential to store the mabl API key
+  1. Make sure that you set the ID of the credentials
+  1. Update the mabl step in each affected job 
+
 ### Change Log
 
+#### v0.0.21 (2 June 2020)
+-   Raised minimum version to 2.121.3
+-   Switched to use Secret text credential kind for storing REST API key
+
+#### v0.0.20 (29 May 2020)
```

Comment:
Are we releasing two versions?

Reply:
No, just one, but you must have clicked on something, because it advanced to 0.0.21-snapshot.